### PR TITLE
BZ1320030: Default Tasks List filters doesn't keep up with user's rol…

### PR DESCRIPTION
**Pull request by nmirasch**
…es changes. Modified admin tab dynamic filter creation.

### CODE REVIEW

**Comment by cristianonicolai** [comment]
+1 thanks @nmirasch

File: jbpm-console-ng-human-tasks/jbpm-console-ng-human-tasks-client/src/main/java/org/jbpm/console/ng/ht/client/editors/taskslist/grid/dash/DataSetTasksListGridPresenter.java [hunk 1]
```diff
@@ -169,14 +169,16 @@ public void getData( final Range visibleRange ) {
                             currentTableSettings.getDataSetLookup().addOperation(filter);
                         }
                     }
-                    if( currentTableSettings.getDataSetLookup().getDataSetUUID().equals(HUMAN_TASKS_WITH_ADMIN_DATASET) ||
-                        currentTableSettings.getDataSetLookup().getDataSetUUID().equals(HUMAN_TASKS_WITH_USER_DATASET)) {
+                    boolean isAdmindDataset = currentTableSettings.getDataSetLookup().getDataSetUUID().equals(HUMAN_TASKS_WITH_ADMIN_DATASET);
 
+                    if( isAdmindDataset ||
```
Comment (csadilek) [inline jbpm-console-ng-human-tasks/jbpm-console-ng-human-tasks-client/src/main/java/org/jbpm/console/ng/ht/client/editors/taskslist/grid/dash/DataSetTasksListGridPresenter.java]: There's a typo here in the variable name.

File: jbpm-console-ng-human-tasks/jbpm-console-ng-human-tasks-client/src/main/java/org/jbpm/console/ng/ht/client/editors/taskslist/grid/dash/DataSetTasksListGridPresenter.java [hunk 2]
```diff
@@ -169,14 +169,16 @@ public void getData( final Range visibleRange ) {
...
+                        currentTableSettings.getDataSetLookup().getDataSetUUID().equals(HUMAN_TASKS_WITH_USER_DATASET)) {
                         if (currentTableSettings.getDataSetLookup().getFirstFilterOp() != null) {
-                            currentTableSettings.getDataSetLookup().getFirstFilterOp().addFilterColumn(getUserGroupFilters()); // add actualowner = idenfier or in my groups
+                            // add  logged user is owner or task in my groups
```
Comment (csadilek) [inline jbpm-console-ng-human-tasks/jbpm-console-ng-human-tasks-client/src/main/java/org/jbpm/console/ng/ht/client/editors/taskslist/grid/dash/DataSetTasksListGridPresenter.java]: Can't say this comment helps me understand what's going on. Could this be refined? Potentially even skipped as the code is clear enough.

**Comment by csadilek** [comment]
Would appreciate fixing the typo and improving the comments, but not a blocker. @cristianonicolai you have permissions to merge now, so feel free to decide/merge.

File: jbpm-console-ng-human-tasks/jbpm-console-ng-human-tasks-client/src/main/java/org/jbpm/console/ng/ht/client/editors/taskslist/grid/dash/DataSetTasksListGridPresenter.java (hunk 2 above)
Comment (nmirasch) [inline jbpm-console-ng-human-tasks/jbpm-console-ng-human-tasks-client/src/main/java/org/jbpm/console/ng/ht/client/editors/taskslist/grid/dash/DataSetTasksListGridPresenter.java]: @csadilek wait for merging. I'm going to fix this comments.Thanks!

**Comment by nmirasch** [comment]
@csadilek @cristianonicolai wait for merging. I'm going to fix this comments.Thanks!

**Comment by nmirasch** [comment]
@cristianonicolai  @csadilek Done!

**Comment by cristianonicolai** [comment]
9fb4497bf3660f2b14caf77939af09d5160ff184

**Pull request closed without merging**